README.md

```diff
@@ -0,0 +1,26 @@
+# Project Theia
+
+## What is it?
+
+Project Theia is a mobile app designed to automatically find and securely
+connect you to free WiFi access points. The app will connect you to nearby
+access points that are both fast and reliable. Once connected, the app will
+use a VPN service, either provided by the user or by us as a subscription,
+to secure your connection and keep you safe.
+
+## What are free WiFi access points?
+
+These are WiFi access points that neither have a password nor captive portal
+and that have not opted out of Project Theia.
+
+## How do I opt out of Project Theia?
+
+For the sake of simplicity, we will be using the same (or extremely similar)
+opt-out convention as Google's WiFi Assistant. That is, you can opt out by doing one or more of the following...
+
+1. Blocking our canary URL;
+2. Securing your access point with a password; or,
+3. Adding `_notheia` to the end of your SSID
```

Comment:
Is this for legal reasons?

Reply:
I doubt we need it for legal reasons. It's not why I included it anyway. I added it just because, if a network admin doesn't want us automatically connecting to their SSID, I'd say that's fair. If they notice a huge spike in data usage that's causing them harm and they suspect it's because of Theia, I think it's only fair that they have a reasonable means of protecting themselves.

With that being said, I doubt Theia will ever be an issue to anyone. It's more about giving network admins the power to have control of how their network is used.